src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -34,6 +57,29 @@ def validate_datetime(string):
     date_format = '%Y-%m-%dT%H:%MZ'
     return datetime.strptime(string, date_format)
 
+def validate_entity(namespace):
+    ''' Converts a list of key value pairs into a dictionary. Ensures that required
+    RowKey and PartitionKey are converted to the correct case and included. '''
+    values = dict(x.split('=', 1) for x in namespace.entity)
+    keys = values.keys()
+    for key in keys:
+        if key.lower() == 'rowkey':
+            val = values[key]
+            del values[key]
+            values['RowKey'] = val
+        elif key.lower() == 'partitionkey':
+            val = values[key]
+            del values[key]
+            values['PartitionKey'] = val
+    keys = values.keys()
+    missing_keys = 'RowKey ' if 'RowKey' not in keys else ''
+    missing_keys = '{}PartitionKey'.format(missing_keys) \
```

Comment:
The space is at the end of 'RowKey '. Otherwise, if only PartitionKey was missing, there'd be an awkward space. If only RowKey is missing, you won't see the DREADED TRAILING WHITESPACE (but Pylint would!) :D  

In short, yes, intended.

---
In reply to: [74457342](https://github.com/Azure/azure-cli/pull/681#discussion_r74457342) [](ancestors = 74457342)